migaku-subtitle-syncer.py

```diff
@@ -62,3 +62,22 @@ def check_if_video_file(filename):
     parser = make_parser()
     args = parser.parse_args(args=unparsed_args)
     result = run(args)
+
+question = QMessageBox.question(
+    None,
+    "Save without \".synced\"",
```

Comment:
The pythonic way to write this would be to use single quotes for the string so you don't have to escape the inner quotes:
```python
'Save without ".synced"',
```

Reply:
Yeah okay I can do it either way, I normally stick to mostly using one or the other and saw you'd used " so stuck with it but either either